Up package version

TimvdHeijden opened the pull request.

TimvdHeijden commented:
@melvinfolkers Is there a good way to test this before merging? After merging we can first use it with a release candidate.

erfannariman commented:
I don't think we should pin versions anymore in packages. I think we should change everything to `>=` so we don't get conflicts anymore, it will be very rare that we run into braking changes, and at that point we can always set an upper limit.

So for this PR I would suggest to keep the original versions and just put `>=` in there.

TimvdHeijden commented:
@erfannariman I don't see your comment anymore, but you mean like this with the `>=`? I was not sure whether its also desired in the `requirements.txt`.

erfannariman commented:
> @erfannariman I don't see your comment anymore, but you mean like this with the `>=`? I was not sure whether its also desired in the `requirements.txt`.

Yes I removed the comment because it would actually introduce breaking changes already, so in this case it's not valid with the Azure packages.

TimvdHeijden commented:
> > @erfannariman I don't see your comment anymore, but you mean like this with the `>=`? I was not sure whether its also desired in the `requirements.txt`.
> 
> Yes I removed the comment because it would actually introduce breaking changes already, so in this case it's not valid with the Azure packages.

Ready for review, limited some packages from above, and replaced the deprecated parameter `line_terminator` with `lineterminator`. All tests succeeded.

TimvdHeijden commented:
@erfannariman or @melvinfolkers can one of you check again?

melvinfolkers commented on df_to_azure/export.py:
```diff
@@ -197,7 +197,7 @@ def upload_to_blob(self):
         )
 
-        data = self.df.to_csv(index=False, sep="^", quotechar='"', line_terminator="\n")
+        data = self.df.to_csv(index=False, sep="^", quotechar='"', lineterminator="\n")
```
side note: Ive created e a new issue  #125 for making this an export to .parquet.

melvinfolkers reviewed (approved):
Approved from my side, but note that some code is written by me also. Maybe @erfannariman  can take a look as well. 

@TimvdHeijden did you run the tests from the test suite?

TimvdHeijden commented:
> Approved from my side, but note that some code is written by me also. Maybe @erfannariman can take a look as well.
> 
> @TimvdHeijden did you run the tests from the test suite?

Yes see my comment from March 17, `All tests succeeded.`

erfannariman commented:
> @erfannariman or @melvinfolkers can one of you check again?

I think we need to set `pandas>=1.5.0` because of the change in argument `lineterminator`.

TimvdHeijden commented:
> > @erfannariman or @melvinfolkers can one of you check again?
> 
> I think we need to set `pandas>=1.5.0` because of the change in argument `lineterminator`.

Done. Tests work, but thats because I already had 1.5.3 locally. and they did not work with pandas 1.4.3. So valid point.
